ios/Video/RCTVideoManager.m

```diff
@@ -80,8 +80,9 @@ @interface RCT_EXTERN_MODULE(RCTVideoManager, RCTViewManager)
 RCT_EXTERN_METHOD(setPlayerPauseState:(nonnull NSNumber *)paused
                  reactTag:(nonnull NSNumber *)reactTag)
 
-RCT_EXTERN_METHOD(presentFullscreenPlayer
-                 reactTag:(nonnull NSNumber *)reactTag)
+RCT_EXTERN_METHOD(presentFullscreenPlayer:(nonnull NSNumber *)reactTag)
+
+RCT_EXTERN_METHOD(dismissFullscreenPlayer:(nonnull NSNumber *)reactTag)
 
 RCT_EXTERN_METHOD(dismissFullscreenPlayer
                  reactTag:(nonnull NSNumber *)reactTag)
```

Comment:
should both of these be here?

Reply:
@jordangrant-stadiumlive 
It looks my mistake.
could you make PR for correct this?

Reply:
posted: https://github.com/react-native-video/react-native-video/pull/3474